src/Compilers/Core/Portable/Generated/Operations.Generated.cs

```diff
@@ -2767,6 +2768,24 @@ internal interface IWithOperation : IOperation
         /// </summary>
         IOperation Value { get; }
     }
+    /// <summary>
+    /// Represents a using variable declaration statement.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IUsingVariableDeclarationOperation : IOperation
```

Comment:
Dumb question.  Why is this a new iop, as opposed to being a bit on IVariableDeclarationGroupOperation ?

Reply:
Resolved